package/gluon-config-mode-theme/sass/gluon.scss

```diff
@@ -184,133 +179,144 @@ ul.tabmenu li {
 	list-style: none;
 	margin: 0;
 	padding: 0;
-}
 
-#topmenu li {
-	display: inline-block;
+	li {
+		display: inline-block;
+	}
 }
 
 #maincontent {
-	padding: 0 1em 2em;
-	max-width: 60em;
-	min-width: 40em;
-	margin: 1em auto;
-}
+	max-width: 60rem;
+	margin: 1rem auto;
+	background-color: rgba(255, 255, 255, 0.333);
+	padding: 0;
+	border-radius: 0.5rem;
+
+	p {
+		margin: 0 1rem 1rem;
+	}
 
-#maincontent p {
-	margin-bottom: 1em;
+	ul {
+		margin-left: 2rem;
+	}
 }
 
 .gluon-section {
-	margin: 0;
+	margin: 0 0 1.25rem;
 	padding: 0;
 	border: none;
-	margin-bottom: 1.3em;
-}
 
-.gluon-section:last-child {
-	margin-bottom: 0.7em;
-}
+	&:last-child {
+		margin-bottom: 0.75rem;
+	}
 
-.gluon-section legend {
-	font-size: 1.4em;
-	font-weight: bold;
-	position: relative;
-	padding: 0;
-	margin-bottom: 0.5em;
-}
+	legend {
+		font-size: 1.5rem;
+		font-weight: bold;
+		position: relative;
+		margin-bottom: 0.5rem;
+		padding: 1rem;
+	}
+
+	h2 {
+		margin: 0 0 0.5rem -0.5rem;
+	}
 
-.gluon-section h2 {
-	margin: 0em 0 0.5em -0.5em;
+	h3 {
+		text-decoration: none;
+		font-weight: bold;
+		color: #555555;
+		margin: 0.25rem;
+		font-size: 100%;
+	}
 }
 
-.gluon-section h3 {
-	text-decoration: none;
-	font-weight: bold;
-	color: #555555;
-	margin: 0.25em;
-	font-size: 100%;
+body.admin-network .gluon-section {
+	margin-bottom: 0;
 }
 
 .gluon-section-descr {
-	margin-bottom: 2em;
+	margin-bottom: 2rem;
 }
 
 .gluon-osm-map {
 	width: 100%;
-	height: 40em;
-	margin-bottom: 1em;
+	height: 40rem;
+	margin-bottom: 1rem;
 }
 
-input::placeholder {
-	color: #aaaaaa;
+#value-id\.1\.6\.map {
```

Comment:
These ID's can't be used for styling, they can change at any time. Which page/element does this even refer to?

Reply:
as far as i remeber atm: thats a textbox appearing next to the map.
because the osm-stuff is placed in another module. i remove that one completely.

Reply:
fixed with ff201e451e91052743b546b797e0206f776a54f0

Reply:
... and 4e66ee27f18518c6c0238023a9b4fc4132c756bd

Reply:
We should add CSS classes to the elements that need better styling, but we can leave that for a later PR.